src/components/ImageUploader.tsx

```
@@ -0,0 +1,194 @@
+        <div
+          onClick={handleClick}
+          onKeyDown={handleKeyDown}
+          onDragOver={handleDragOver}
+          onDragLeave={handleDragLeave}
+          onDrop={handleDrop}
+          className={cn(
+            "relative flex h-[120px] w-[120px] cursor-pointer flex-col items-center justify-center overflow-hidden rounded-lg border-none",
+          )}
+          role="button"
+          tabIndex={0}
+          aria-label={label || "이미지 교체"}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)